sql/core/src/test/resources/decimal.csv

```diff
@@ -0,0 +1,4 @@
+decimal
+2160273033000006010010000035858
```

Comment:
Maybe you meant adding some more data that can be inferred as `LongType` or `DoubleType` in the test file to check type inference correctly?

I updated the test data and this produces the schema below:

```bash
root
 |-- decimal-precision: decimal(20,0) (nullable = true)
 |-- decimal-scale: decimal(18,17) (nullable = true)
```